saleor/plugins/webhook/tasks.py

```diff
@@ -219,60 +222,89 @@ def send_webhook_using_http(
 
 
 def send_webhook_using_aws_sqs(target_url, message, domain, signature, event_type):
-    parts = urlparse(target_url)
-    region = "us-east-1"
-    hostname_parts = parts.hostname.split(".")
-    if len(hostname_parts) == 4 and hostname_parts[0] == "sqs":
-        region = hostname_parts[1]
-    client = boto3.client(
-        "sqs",
-        region_name=region,
-        aws_access_key_id=parts.username,
-        aws_secret_access_key=(
-            unquote(parts.password) if parts.password else parts.password
-        ),
-    )
-    queue_url = urlunparse(
-        ("https", parts.hostname, parts.path, parts.params, parts.query, parts.fragment)
-    )
-    is_fifo = parts.path.endswith(".fifo")
-
-    msg_attributes = {
-        "SaleorDomain": {"DataType": "String", "StringValue": domain},
-        "EventType": {"DataType": "String", "StringValue": event_type},
-    }
-    if signature:
-        msg_attributes["Signature"] = {"DataType": "String", "StringValue": signature}
-
-    message_kwargs = {
-        "QueueUrl": queue_url,
-        "MessageAttributes": msg_attributes,
-        "MessageBody": message.decode("utf-8"),
-    }
-    if is_fifo:
-        message_kwargs["MessageGroupId"] = domain
-    with catch_duration_time() as duration:
-        response = client.send_message(**message_kwargs)
-        return WebhookResponse(content=response, duration=duration())
+    try:
```

Comment:
Can't we put the `try..except` around the code which we exactly know can throw a specific exception? It would be a bit prettier and easier to debug when the exception occurs I think.